CommonControls/Common/SaveHelper.cs

```diff
@@ -169,5 +173,56 @@ static string IndexedFilename(string stub, string extension)
             } while (File.Exists(filename));
             return filename;
         }
+
+        static void EnsureTexconvExists()
+        {
+            var texconvPath = $"{DirectoryHelper.TempDirectory}\\texconv.exe";
+
+            if (!File.Exists(texconvPath))
+            {
+                using var stream = Assembly.GetExecutingAssembly().GetManifestResourceStream("View3D.Content.Other.texconv.exe");
+                using var fStream = new FileStream(texconvPath, FileMode.OpenOrCreate);
+                stream!.CopyTo(fStream);
+            }
+        }
+
+        public static void SavePNGTextureAsDDS(string pngFilePath)
+        {
+            var texconvPath = $"{DirectoryHelper.TempDirectory}\\texconv.exe";
+
+            EnsureTexconvExists();
+
+            using var pProcess = new System.Diagnostics.Process();
+            pProcess.StartInfo.FileName = texconvPath;
+            pProcess.StartInfo.Arguments =
+                $"-sRGBi -f BC7_UNORM_SRGB -y -o \"{Path.GetDirectoryName(pngFilePath)}\" \"{pngFilePath}\"";
```

Comment:
is there not different formats depending on what type of texture (diffuse, normal, specular?)  it is?